docs/devGuide/migratingToTypeScript.md

```diff
@@ -116,13 +101,49 @@ You can see these pull requests to observe the finished migration works:
 
 - [#1877: Adopt TypeScript for core package](https://github.com/MarkBind/markbind/pull/1877)
 
+## Import/Export Syntax Reference
+
+Here is the reference on changing the import/export statements from CommonJS syntax into those that TypeScript support: TypeScript equivalent syntax, and ES6 syntax.
+
+The TypeScript equivalent syntax is designed to compile to the exact same code as CommonJS. You can read more about this syntax in the [official documentation](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require).
+
+The ES6 syntax is the newer JavaScript syntax that TypeScript recommends, but in certain cases it does not compile to the exact same code as CommonJS.
+
+In the tables below, a "thing" is defined as a JavaScript construct that can be exported, such as constants, functions, classes, etc. In the ES6 syntax, a "thing" can also be TypeScript constructs like types, interfaces, enums, etc.
+
+!!**Exports**!!
+
+| What to Export              | CommonJS                         | TypeScript equivalent   | ES6                   |
+|-----------------------------|----------------------------------|-------------------------|-----------------------|
+| One thing only              | `module.exports = x`             | `export = x`            | `export default x`*   |
```

Comment:
👀  would footnotes look better / ok? (on the "*" and "^"'s)

could also try bolding it if not to make it stand out from normal text a little bit more

Reply:
I'm not sure on the footnotes. I feel like these are important info that readers should understand, and putting it in our footnotes component hides it from view unless the reader intentionally hovers to it.

My intention for the symbols is to contextualize which syntax the info is referring to. In a way, it's like footnotes, but I just wanted to make the content more prominent than actual footnotes.

Reply:
formatting nit to make the asterixes appear more prominent:

`<span style="font-weight:bold;">&nbsp;*</span>`

could also stick to `*` / one symbol to make it consistent.

Reply:
:+1: I'll take the `^` symbol as it's more visible than `*`